scripts/rebuild-ui.sh

```diff
@@ -7,7 +7,11 @@ set -euxf -o pipefail
 
 cd jaeger-ui
 
-git fetch --all --unshallow --tags
+git fetch --all --tags
+is_shallow_repo=$(git rev-parse --is-shallow-repository)
+if [[ "$is_shallow_repo" == "true" ]]; then
+    git fetch --unshallow
+fi
```

Comment:
I think you want this in the reverse order
```suggestion
if [[ "$(git rev-parse --is-shallow-repository)" == "true" ]]; then
    git fetch --unshallow
fi
git fetch --all --tags
```